### PR TITLE
Add libadios-dev packages to build clients for package 'openPMD-api'

### PR DESCRIPTION
**Pull request by ax3l**
Re: https://github.com/piwheels/packages/issues/23

### CODE REVIEW

**Comment by ax3l** [comment]
Hi @bennuttall, can this be merged? :)

File: deploy_slave.sh [hunk 1]
```diff
@@ -64,7 +64,7 @@ apt -y install vim ssh-import-id tree byobu htop pkg-config cmake time \
     libtiff5-dev libx11-6 libx11-dev xfonts-base xfonts-100dpi xfonts-75dpi \
     xfonts-cyrillic fluid-soundfont-gm libsystemd-dev libusb-1.0-0-dev \
     libudev-dev libopus-dev libvpx-dev libc-bin libavdevice-dev \
-    libavfilter-dev libavutil-dev lsb-release
+    libadios-dev libavfilter-dev libavutil-dev lsb-release
```
Comment (ax3l) [inline deploy_slave.sh]: hm, is this even in raspbian?
It's on all official debian architectures: https://packages.debian.org/buster/libadios-dev

raspbian arch? http://archive.raspberrypi.org/debian/pool/main/

Comment (ax3l) [reply deploy_slave.sh]: Hm, according to this index it should be there, somewhere: https://archive.raspbian.org/raspbian/dists/buster/main/binary-armhf/

**Comment by ax3l** [comment]
ping @bennuttall @waveform80

**Comment by waveform80** [comment]
Yup, this can be merged - I'll just install it manually on the build-slaves

**Comment by ax3l** [comment]
Thanks a lot! :)

**Pull request merged**